index.js

```diff
@@ -3,53 +3,225 @@ const state = {
     {
       id: "001-beetroot",
       name: "beetroot",
-      price: 0.35
+      price: 0.35,
+      type: 'vegetable'
     },
     {
       id: "002-carrot",
       name: "carrot",
-      price: 0.35
+      price: 0.35,
+      type: 'vegetable'
     },
     {
       id: "003-apple",
       name: "apple",
-      price: 0.35
+      price: 0.35,
+      type: 'fruit'
     },
     {
       id: "004-apricot",
       name: "apricot",
-      price: 0.35
+      price: 0.35,
+      type: 'fruit'
     },
     {
       id: "005-avocado",
       name: "avocado",
-      price: 0.35
+      price: 0.35,
+      type: 'fruit'
     },
     {
       id: "006-bananas",
       name: "bananas",
-      price: 0.35
+      price: 0.35,
+      type: 'fruit'
     },
     {
       id: "007-bell-pepper",
       name: "bell pepper",
-      price: 0.35
+      price: 0.35,
+      type: 'fruit'
     },
     {
       id: "008-berry",
       name: "berry",
-      price: 0.35
+      price: 0.35,
+      type: 'fruit'
     },
     {
       id: "009-blueberry",
       name: "blueberry",
-      price: 0.35
+      price: 0.35,
+      type: 'fruit'
     },
     {
       id: "010-eggplant",
       name: "eggplant",
-      price: 0.35
+      price: 0.35,
+      type: 'fruit'
     }
   ],
-  cart: []
-};
\ No newline at end of file
+  cart: [],
+  filteredByFruit: false,
+  sortedAToZ: false
+};
+
+function createStoreListItem(item) {
+  const ul = document.querySelector('.store--item-list')
+  const li = document.createElement('li')
+  const div = document.createElement('div')
+  const img = document.createElement('img')
+  const button = document.createElement('button')
+  
+  
+  div.setAttribute('class', 'store--item-icon')
+  img.setAttribute('src', `assets/icons/${item.id}.svg`)
+  img.setAttribute('alt', `${item.name}`)
+  button.innerText = 'Add to Cart'
+  button.addEventListener('click', () => {
+    item.quantity++
+    if(state.cart.includes(item)) {
+      render()
+      return
+    } 
+    state.cart.push(item)
+    render()
+  })
+
+  div.append(img)
+  li.append(div, button)
+  ul.append(li)
+}
+
```

Comment:
i really like the way you have structured the code here in the first 2 functions  - to select the list and cart and create elements to go inside those. it all reads v clearly